Core/AppTrackingProtectionListViewModel.swift

```diff
@@ -91,6 +97,19 @@ public class AppTrackingProtectionListViewModel: NSObject, ObservableObject, NSF
         
         return relativeDate
     }
+    
+    public func format(timestamp: Date) -> String {
+        let date = Date()
+        let timestampInterval = timestamp.timeIntervalSinceReferenceDate
+        let dateInterval = date.timeIntervalSinceReferenceDate
+        if fabs(dateInterval - timestampInterval) < 1 {
+            // Can't access UserText from Core. To prevent handling the localized string "in 0 seconds"
+            // return "0" here and replace it with UserText on the view side.
+            return "0"
```

Comment:
What do you think about having this function return an optional, and in this case returning `nil`? Right now the value of `"0"` feels a bit magic and requires the caller to know about an implementation detail of this function - if it ever changed, we'd end up with a bug on the client side but have no way for the compiler to tell us about it.

Reply:
Yeah that makes way more sense 🤦‍♂️